src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calcArea() {
+        return Math.PI * Math.sqrt(radius);
+    }
+
+    @Override
+    public void draw() {
+        double area = calcArea();
+        String colorString = color.toString();
```

Comment:
1. in all draw methods these variables may be easily inlined into printf command, they are short and has proper naming to be undertandable
2. please use .name() method to get String representation of ENUM:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants